tests/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ if (UNIX OR ADD_WINDOWS_ENCLAVE_TESTS OR USE_CLANGW)
         add_subdirectory(attestation_plugin)
         add_subdirectory(backtrace)
         add_subdirectory(bigmalloc)
+        add_subdirectory(child_process)
```

Comment:
This test depends on the host process invoking `fork()` which is not supported on Windows, I don't think it will work cross-platform. Can we move this test down to line 87 as a Linux-specific test?

Reply:
I have move this project to Linux-specific test. Thanks.